### DISCUSSION TITLE
Failed to send magic link: failed to make magiclink request: Error sending magic link

### DISCUSSION THREAD

**Discussion by jthathapudi** [supabase/supabase#27043] [Questions] [answered]
I have configured my own STMP server(Zoho), but still I am getting the error ''
Failed to send magic link: failed to make magiclink request: Error sending magic link
'

Any help is highly appreciated

**Comment by GaryAustin1** [comment, marked as answer]
Look in the Auth logs on the Dashboard to see if there is an error.  
What are you doing that gets the error?

**Reply by jthathapudi** [reply]
Looks like some issue with SMPT setting. It is fixed and working now. Thank you
